packages/release-brancher/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "release-brancher",
+  "version": "1.0.0",
+  "description": "Cut release branches",
+  "bin": "./build/src/bin/release-brancher.js",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "files": [
+    "build/src"
+  ],
+  "scripts": {
+    "compile": "tsc -p .",
+    "start": "probot run ./build/src/release-brancher.js",
+    "start:local": "node ./build/src/local.js",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal mocha --exit build/test",
+    "fix": "gts fix",
+    "lint": "gts check"
+  },
+  "dependencies": {
+    "@octokit/rest": "^18.0.1",
+    "code-suggester": "^1.9.1",
+    "yargs": "^17.0.0-candidate.6",
+    "js-yaml": "^4.0.0"
+  },
+  "devDependencies": {
+    "@types/js-yaml": "^4.0.0",
+    "@types/mocha": "^8.0.0",
+    "@types/node": "^12.6.1",
+    "@types/pino": "^6.3.6",
```

Comment:
Are we using pino?

Reply:
No, but I think I added it because code-suggester uses it internally and it was causing type issues. I don't think types should be a runtime dependency, but I'm not sure why this was failing (perhaps types leaking)?